src/govuk/common.mjs

```diff
@@ -1,4 +1,5 @@
 import './vendor/polyfills/Element/prototype/dataset.mjs'
+import './vendor/polyfills/Element/prototype/closest.mjs'
```

Comment:
Does the polyfill definitely go here?

```suggestion
```

Reply:
Happy to merge with this line, we're going to address it before the next release under https://github.com/alphagov/govuk-frontend/issues/2907